pytorch_lightning/callbacks/xla_stats_monitor.py

```diff
@@ -73,10 +73,10 @@ def on_train_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule")
         if not trainer.loggers:
             raise MisconfigurationException("Cannot use XLAStatsMonitor callback with Trainer that has no logger.")
 
-        if isinstance(trainer.accelerator, TPUAccelerator):
+        if not isinstance(trainer.accelerator, TPUAccelerator):
```

Comment:
This change is not reflected in the tests I believe. Can you double check that the error message gets correctly caught/asserted? Thanks

Reply:
looks like there is no test for it.

Reply:
I added a new test case in `tests/deprecated_api/test_remove_1-7.py::test_xla_stats_monitor_tpu_not_used` to validate it.